components/Admin/Admin.js

```diff
@@ -22,35 +24,67 @@ class Admin extends React.Component {
 
   handleSearch = (event) => {
     const searchTerm = event.currentTarget.value
-    let { locations, filteredLocations } = this.state
+    let { locations, matchingSearchLocations } = this.state
 
     if (searchTerm) {
-      filteredLocations = locations.filter(location =>
+      matchingSearchLocations = locations.filter(location =>
```

Comment:
Am I? What does `let { foo } = this.state` do? Doesn't it just do `let foo = this.state.foo`?  If so, saying `foo = 7` after that doesn't update `this.state.foo` does it?